### DISCUSSION TITLE
Can I restore scroll position in a scrollable container?

### DISCUSSION THREAD

**Discussion by rubenen** [TanStack/query#971] [General]
I'm using the infinite query hook to load a list of posts. As I scroll down more posts are loaded and I can navigate to a post and back again (using the browser back button) to the list and my scroll position is retained. 

However if my list of posts is in a scrollable container I lose my scroll position as I navigate back and forth. Is there any solution to this?

Demo: [https://u1q0r.csb.app/](https://u1q0r.csb.app/)

**Comment by nubpro** [comment]
Wondering is there a solution to this?
